src/gpuarray_blas_cuda_cublas.c

```diff
@@ -573,11 +573,11 @@ static int hgemm3D(cb_order order, cb_transpose transA, cb_transpose transB,
   GA_CUDA_EXIT_ON_ERROR(ctx, cuda_wait(C, CUDA_WAIT_ALL));
   err = cublasHgemmStridedBatched(h->h,
 				  convT(transA), convT(transB),
-				  M, N, K, &halpha,
+				  M, N, K, (__half *)&halpha,
```

Comment:
There is a conversion above with a function.  This is just a cast so the compiler is happy.